benchmarks/ModelingToolkit/RCCircuit.jmd

```diff
@@ -0,0 +1,169 @@
+  @show n, ss_times[i, :], times[i, :]
+end
+
+N = vcat([5, 10, 20, 40, 60, 80, 160, 320, 480, 640, 800], 1_000:1_000:10_000);
+
+# max size we test per method
+max_sizes = [1000, 4_000, 10_000, 10_000, 10_000, 10_000, 10_000];
+
+# NaN-initialize so Makie will ignore incomplete
+ss_times = fill(NaN, length(N), 3);
+times = fill((NaN,NaN,NaN), length(N), length(max_sizes));
+
+@time run_and_time!(ss_times, times, max_sizes, 1, 4); # precompile
+
+for (i, n) in enumerate(N)
+  run_and_time!(ss_times, times, max_sizes, i, n)
+end
+
+f = Figure();
+let ax = Axis(f[1, 1]; title="Structural Simplify Time")
+  for (j, label) in enumerate(("MTK", "JSIR-Scalar", "JSIR-Loop"))
+    ts = @view(ss_times[:, j])
+    lines!(N, ts; label)
+  end
+  axislegend(ax, position = :lt)
+end
+for (i, timecat) in enumerate(("ODEProblem + f!", "Run", "Solve"))
+  title = timecat * " Time"
+  ax = Axis(f[i+1, 1]; title)
+  for (j, label) in enumerate(("MTK", "JSIR - Scalar - Julia", "JSIR - Scalar - C", "JSIR - Scalar - LLVM", "JSIR - Loop - Julia", "JSIR - Loop - C", "JSIR - Loop - LLVM"))
+    ts = getindex.(@view(times[:, j]), i)
+    lines!(N, ts; label)
+  end
+  axislegend(ax, position = :lt)
+end
+f
+```
+All three backends compiled more quickly with loops, but the C and LLVM backends are so much quicker to compile than the Julia backend that this made much less difference for them.
+The impact on runtime was more varied.
+
+## Appendix
+
+```julia, echo = false
+using SciMLBenchmarks
+SciMLBenchmarks.bench_footer(WEAVE_ARGS[:folder],WEAVE_ARGS[:file])
+```
+
+## Time OpenModelica
+
+```julia
+using OMJulia, CSV, DataFrames
+mod = OMJulia.OMCSession();
+
+OMJulia.sendExpression(mod, "getVersion()")
+
+modelicafile = joinpath(@__DIR__, "RC_Circuit.mo")
+
+resultfile = joinpath(@__DIR__, "RC_Circuit_modelica_res.csv")
+
+@show "Start OpenModelica Timings"
+
+# TODO: Sync these test cases with the N used in MTK: 
+N_x = [5, 10, 20, 40, 60, 80, 160, 320, 480, 640, 800, 1000, 2000, 3000, 4000, 5000, 6000, 7000, 8000, 9000, 10000, 20000]
+
+for N in N_x
+    @show N
+    totaltime = @elapsed res = begin
+        ModelicaSystem(mod, modelicafile, "RC_Circuit.Test.RC_Circuit_MTK_test_$N")
+        sendExpression(mod, "simulate(RC_Circuit.Test.RC_Circuit_MTK_test_$N)")
+    end
+    #runtime = res["timeTotal"]
+    @assert res["messages"][1:11] == "LOG_SUCCESS"
+    @show totaltime
+    # total_times[i, 5] = totaltime TODO: Add to comparison table
+end
+
+OMJulia.quit(mod) 
+```
+
+Dymola requires a license server and thus cannot be hosted. This was run locally for the
+following times:
+
+```julia
+translation_and_total_times = [
+7.027, 7.237
+11.295, 11.798
+16.681, 17.646
+22.125, 23.839
+27.529, 29.82
+33.282, 36.622
+39.007, 43.088
+44.825, 51.601
+50.281, 56.676
+] # TODO: I will add other times once the Dymola license server is back up.
+#total_times[:, 6] = translation_and_total_times[1:length(N_x),2]
+```
```

Comment:
Add the total times plots for all:

https://github.com/SciML/SciMLBenchmarks.jl/pull/949/files#diff-c0aa5047a0078d961dd9b04e0cb2fac675f99d8c0b809efeb64b1ffc4bd2b8b3R420-R432